### PR TITLE
RELATED: CAT-336 Extend DialogBase onKeyDown submit disable

### PR DESCRIPTION
**Pull request by Tulean**
JIRA: CAT-336

---

Supported PR commands:

| Command         | Description            |
| --------------- | ---------------------- |
| `ok to test`    | Re-run standard checks |
| `extended test` | BackstopJS tests       |

---

# PR Checklist

-   [ ] commit messages adhere to the [commit message guidelines](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#what-should-the-commits-look-like)
-   [ ] review was done by a Code owner [if necessary](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-tell-if-my-pull-request-needs-approval-by-a-code-owner) (if you think it is not necessary, explain the reasoning in the description or in a comment)
-   [ ] `check` passes
-   [ ] `check-extended` passes
-   [ ] `rush change` [was run if applicable](https://github.com/gooddata/gooddata-ui-sdk/blob/master/docs/contributing.md#how-do-i-describe-my-changes-for-the-changelog)

### CODE REVIEW

**Comment by Tulean** [comment]
extended test

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1808/ : SUCCESS in 5s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3417/ : SUCCESS in 21m 06s

**Comment by gdgate** [comment]
Build succeeded (check-extended pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-oss-check/1809/ : SUCCESS in 9s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-backstop-tests/546/ : SUCCESS in 34m 16s

File: libs/sdk-ui-kit/src/Dialog/DialogBase.tsx [hunk 1]
```diff
@@ -15,12 +15,13 @@ const checkKeyHandler = (event: React.KeyboardEvent, keyCode: number, handler: (
     }
 };
 
-const isTextInputComponent = ({ target }: React.KeyboardEvent): boolean => {
+const inputComponentHandler = ({ target }: React.KeyboardEvent): boolean => {
```
Comment (no23reason) [inline libs/sdk-ui-kit/src/Dialog/DialogBase.tsx:18]: The new name is not ideal, can we do something like `shouldSubmitOnEnterPress` or something like that? The key thing here is the name should suggest this returns a boolean.

**Review by no23reason — APPROVED** [review]
LGTM

**Comment by gdgate** [comment]
Build in pipeline check aborted.

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-fossa-scanning-zuul-el7/581/ : SUCCESS in 40s (non-voting)
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-sdk-merge-compliance-check/842/ : SUCCESS in 27s
- https://checklist.intgdc.com/job/gooddata-ui-sdk/job/gooddata-ui-sdk-unit-tests/3423/ : SUCCESS in 14m 50s

**Pull request merged**